packages/design-system-react/src/components/temp-components/Maskicon/Maskicon.tsx

```diff
@@ -0,0 +1,41 @@
+import DOMPurify from 'dompurify';
+import React, { useEffect, useState } from 'react';
+
+import { MaskiconProps } from './Maskicon.types';
+import { getMaskiconSVG } from './Maskicon.utilities';
+
+export const Maskicon = ({
+  address,
+  size = 32,
+  style,
```

Comment:
Could we add a classname prop so we are explicit about applying classnames to the outer and should it use twMerge like it mentions in the docs https://github.com/MetaMask/metamask-design-system/pull/554/files#diff-cbe3dc9cbce1196b288ce114a6ac08ba8aa4157c6c942d20285ed3b62909c3e6R40?

Reply:
ComponentProps<'div'> should already have a `className` prop and it's being spread to the outer div already?

Reply:
Do we want to add `twMerge` support here?

Reply:
I dont think its needed so I removed the twmerge line from the doc